src/production/transformation/__tests__/__snapshots__/condition.unwrapper.test.ts.shapshot.json

```diff
@@ -45,5 +45,44 @@
   "41d36129": "alert('no')",
   "573e3511": "alert('no')",
   "07044b600": "console.log(\"yes\")",
-  "f91ebe7": "function test(){\n          return;\n      }\n      "
+  "f91ebe7": "function test(){\n          return;\n      }\n      ",
```

Comment:
what's this? ;-) why has it been updated?

Reply:
These files are Jest snapshot files. Automatically occurred when I added tests....

Reply:
We can turn this off in Jest config. Are you familiar with Jest Snapshots feature? https://jestjs.io/docs/en/snapshot-testing

More used for UI code.